framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/WorkbenchKieServerScenarioImpl.java

```diff
@@ -49,20 +49,20 @@
 
     private WorkbenchDeploymentImpl workbenchDeployment;
     private KieServerDeploymentImpl kieServerDeployment;
-    private boolean deployPrometheus;
+    private final boolean deployPrometheus;
     private PrometheusDeployment prometheusDeployment;
 
     private static final Logger logger = LoggerFactory.getLogger(WorkbenchKieServerScenarioImpl.class);
 
-    public WorkbenchKieServerScenarioImpl(KieApp kieApp, boolean deployPrometheus) {
+    public WorkbenchKieServerScenarioImpl(final KieApp kieApp, final boolean deployPrometheus) {
```

Comment:
It's not needed to add the final in the method arguments from Java 8 (it's done by us automatically).